project/config/settings/common.py

```diff
@@ -52,6 +52,7 @@
     'rest_framework',
     'rest_framework.authtoken',
     'django_markdown',
+    'django_jinja',
```

Comment:
Really sure: "from django.template.loader import render_to_string" isn't enough? Is django_jinja worth the extra dependency?